docs/system-management/agent/Ch_SysMgmtAgent.rst

```diff
@@ -5,4 +5,206 @@ System Management Agent (SMA)
 ============
 Introduction
 ============
-More documentation coming soon.
\ No newline at end of file
+
+While the SMA serves several purposes, it is to be considered, first and foremost, as the single connection point of management control for an EdgeX instance. As such, the API calls related to system management are defined so as to interface with the SMA.
+
+=====================
+Examples of API Calls
+=====================
+
+To get an appreciation for some SMA API calls in action, it will be instructive to look at what responses the SMA provides to the caller, for the respective calls (Notice, too, the error messages returned by the SMA, should it encounter a problem). Thus, consider the following calls that the SMA handles:
+
+* Metrics of a service
+* Configuration of a service
+* Start a service
+* Stop a service
+* Restart a service
+* Health check on a service
+
+Let's look at the preceding calls (aka requests), one-by-one, in the following sections.
+
+* Metrics of a service
+
+Example request:
+/api/v1/metrics/edgex-core-command,edgex-core-data
+
+Corresponding response, in JSON format:
+{
+   "Metrics":{
```

Comment:
If you prefix your JSON with:
```
.. code-block:: json

```
It will do pretty formatting for you. Be sure to include a blank line after the code-block statement, and indent every line in the block, then leave another blank line at the end of it.